test/e2e_node/topology_manager_test.go

```diff
@@ -766,6 +920,37 @@ func runTopologyManagerTests(f *framework.Framework) {
 		// Delete state file to allow repeated runs
 		deleteStateFile()
 	})
+
+	ginkgo.It("run the Topology Manager pod scope alignment test suite", func() {
+		sriovdevCount := detectSRIOVDevices()
+		numaNodes := detectNUMANodes()
+		coreCount := detectCoresPerSocket()
+
+		if numaNodes < minNumaNodes {
+			e2eskipper.Skipf("this test is intended to be run on a multi-node NUMA system")
+		}
+		if coreCount < minCoreCount {
+			e2eskipper.Skipf("this test is intended to be run on a system with at least 4 cores per socket")
```

Comment:
Good.
Better:
`e2eskipper.Skipf("this test is intended to be run on a system with at least %d cores per socket", minCoreCount)`
(will fix other instances later)

Reply:
You're right, I changed it.